x-pack/filebeat/docs/inputs/input-httpjson.asciidoc

```diff
@@ -199,37 +199,37 @@ The content inside the brackets `[[` `]]` is evaluated. For more information on
 
 Some built-in helper functions are provided to work with the input state inside value templates:
 
-- `add`: adds a list of integers and returns their sum.
+- `add`: Adds a list of integers and returns their sum.
```

Comment:
These lists sit in an precarious position. It's true that they are all full-stop terminated, but they are part of a complete sentence that starts the line above this. Strictly speaking they should be comma or semicolon separated, but I think just leaving them as is would be better.

Reply:
Can the capitalisation of this be reverted?

Reply:
Done. I apologize for the delay in getting back to you.